### PR TITLE
CPDP-1858

### PR DESCRIPTION
**Pull request by sam-miele**
Updates content for Mixed use guidance for period of admin (Single and multiple), single mixed use market value capture page, single mixed use disposal page [JIRA](https://jira.tools.tax.service.gov.uk/browse/CPDP-1858)

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -4088,6 +4113,8 @@ singleMixedUseDisposalsAcquisitionPrice.personalRep.details.p3=If the person sha
 singleMixedUseDisposalsAcquisitionPrice.capacitor.details.p3=If the person shared any of the costs, they can only claim their share.
 singleMixedUseDisposalsAcquisitionPrice.agent.details.p3=If your client shared any of the costs, they can only claim their share.
 singleMixedUseDisposalsAcquisitionPrice.trust.details.p3=If the trust shared any of the costs, they can only claim its share.
+singleMixedUseDisposalsAcquisitionPrice.personalRepInPeriodOfAdmin.details.p3=If the person shared any of the costs, you can only include the person's share.
```
Comment (AndyHWChung) [inline conf/messages:4116]: there's a straight quote here and below

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/single_mixed_use_acquisition_price.scala.html [hunk 2]
```diff
@@ -43,15 +45,17 @@
...
 )(implicit request: RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
 
 @key = @{ "singleMixedUseDisposalsAcquisitionPrice" }
 @isAgent = @{ request.userType.contains(UserType.Agent) }
 @userKey = @{
     representativeType match {
         case Some(PersonalRepresentative) => ".personalRep"
-        case Some(PersonalRepresentativeInPeriodOfAdmin) => ".personalRep"
+        case Some(PersonalRepresentativeInPeriodOfAdmin) if(!isAgent) => ".personalRepInPeriodOfAdmin"
+        case Some(PersonalRepresentativeInPeriodOfAdmin) if(isAgent) => ".personalRepInPeriodOfAdmin.agent"
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/single_mixed_use_acquisition_price.scala.html:58]: does the summary template also need updating to include these new user key?

**Comment by AndyHWChung** [comment]
did you double check the multiple disposal journey to see if the messages for the disposal price and market value page for period of admin/agent of period of admin are right?

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/single_mixed_use_acquisition_price.scala.html (hunk 2 above)
Comment (sam-miele) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/single_mixed_use_acquisition_price.scala.html:58]: adding now

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**